bool speed70
bool stop_sign
